webui/react/src/pages/ResourcepoolDetail.tsx

```diff
@@ -197,4 +200,10 @@ const ResourcepoolDetail: React.FC = () => {
   );
 };
 
-export default ResourcepoolDetail;
+const ResourcepoolDetailPage: React.FC = () => (
+  <Suspense fallback={<Page loading />}>
```

Comment:
Thanks to @keita-determined for linking me to some resources on how to use the new Suspense api in React 18. It makes some of these trickier loading states easier.

If you're wondering why I didn't use Suspense everywhere; it has a pretty big limitation in that a component can't manage its own loading state, you need to wrap the component usage in `<Suspense>` (and of course not all loading states should prevent rendering).